xml/System.Security.Cryptography.Pkcs/Rfc3161TimestampRequest.xml

```diff
@@ -47,15 +54,34 @@
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="requestedPolicyId">To be added.</param>
-        <param name="nonce">To be added.</param>
-        <param name="requestSignerCertificates">To be added.</param>
-        <param name="extensions">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="data">The data to timestamp, which will be hashed by this method.</param>
+        <param name="hashAlgorithm">The hash algorithm to use with this timestamp request.</param>
+        <param name="requestedPolicyId">The Object Identifier (OID) for a timestamp policy the Timestamp Authority (TSA) should use, or <see langword="null" /> to express no preference.</param>
+        <param name="nonce">An optional nonce (number used once) to uniquely identify this request to pair it with the response. The value is interpreted as an unsigned big-endian integer and may be normalized to the encoding format.</param>
+        <param name="requestSignerCertificates">
+          <see langword="true" /> to indicate the Timestamp Authority (TSA) must include the signing certificate in the issued timestamp token; otherwise, <see langword="false" />.</param>
+        <param name="extensions">An optional collection of extensions to include in the request.</param>
+        <summary>Create a timestamp request by hashing the provided data with a specified algorithm.</summary>
+        <returns>An <see cref="T:System.Security.Cryptography.Pkcs.Rfc3161TimestampRequest" /> representing the chosen values.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Cryptographic timestamps are always applied to a combination of the hash of data and the algorithm identifier for the hash algorithm used to hash the data,
+this method is a convenient alternative to <xref:System.Security.Cryptography.Pkcs.Rfc3161TimestampRequest.CreateFromHash(System.ReadOnlyMemory{System.Byte},System.Security.Cryptography.HashAlgorithmName,System.Security.Cryptography.Oid,System.Nullable{System.ReadOnlyMemory{System.Byte}},System.Boolean,System.Security.Cryptography.X509Certificates.X509ExtensionCollection)> when the data has not already been hashed.
+
+The nonce value in this method is treated as an unsigned, big-endian integer.
+This method normalizes the input to a BER INTEGER encoding before assigning the nonce value.
+The normalized value is returned by the <xref:System.Security.Cryptography.Pkcs.Rfc3161TimestampRequest.GetNonce> method.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="hashAlgorithm" />.<see cref="P:System.Security.Cryptography.HashAlgorithmName.Name" /> is <see langword="null" /> or <see cref="F:System.String.Empty" />.</exception>
```

Comment:
You can see how this renders at https://review.docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.pkcs.rfc3161timestamprequest.createfromdata?view=netcore-3.1&branch=pr-en-us-3703#exceptions. It's a little hard to read with all the formatting.

![image](https://user-images.githubusercontent.com/24882762/71756131-e735bf80-2e42-11ea-84f3-f7fdac17357b.png)

Reply:
That's fair, I'll accept anything here.  I thought I copied this from IncrementalHash, but apparently there only Name is a link: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.incrementalhash.createhash?view=netframework-4.8#exceptions